ios/COVIDSafePaths/AppDelegate.m

```diff
@@ -80,15 +81,25 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
 }
 
 - (void)applicationWillTerminate:(UIApplication *)application {
-  UILocalNotification *notification = [[UILocalNotification alloc] init];
-  notification.fireDate = [NSDate dateWithTimeIntervalSinceNow:0];
-  notification.alertTitle = @"COVID Safe Paths Was Closed";
-  notification.alertBody = @"COVID Safe Paths requires to be running.";
-  notification.timeZone = [NSTimeZone defaultTimeZone];
-  notification.soundName = UILocalNotificationDefaultSoundName;
-  notification.applicationIconBadgeNumber = 0;
-
-  [RNCPushNotificationIOS didReceiveLocalNotification:notification];
+  //Show local notifiation at first time only.
+  if(![[NSUserDefaults standardUserDefaults] boolForKey:@"NotificationAtFirstTimeOnly"]) {
+    if ([[UIApplication sharedApplication] respondsToSelector:@selector(currentUserNotificationSettings)]){
+      //Checking local notification permission or not.
+      UIUserNotificationSettings *grantedSettings = [[UIApplication sharedApplication] currentUserNotificationSettings];
+      if (grantedSettings.types != UIUserNotificationTypeNone){
+        // Set local notification.
+        UILocalNotification *_localNotification = [[UILocalNotification alloc] init];
+        _localNotification.fireDate = [NSDate dateWithTimeIntervalSinceNow:5];
+        _localNotification.timeZone = [NSTimeZone defaultTimeZone];
+        _localNotification.alertTitle = @"COVID Safe Paths";
```

Comment:
Thanks, and now can you sync with `develop` branch to add these strings into en.lproj and localize them?

e.g.

https://github.com/tripleblindmarket/covid-safe-paths/blob/5f2f131b57018e7ee849e829a028a3e78b301755/ios/en.lproj/Localizable.strings#L2-L5

https://github.com/tripleblindmarket/covid-safe-paths/blob/5f2f131b57018e7ee849e829a028a3e78b301755/ios/COVIDSafePaths/AppDelegate.m#L83-L84

Reply:
Ok. I will update.

Reply:
@tstirrat Merged my branch developer into ganesh-ios. You can check two file AppDelegate.m and  en.lproj/Localizable.strings only.